kong/clustering/services/sync/rpc.lua

```diff
@@ -66,6 +66,17 @@ end
 function _M:init_cp(manager)
   local purge_delay = manager.conf.cluster_data_plane_purge_delay
 
+  -- CP
+  -- Method: kong.sync.v2.notify_validation_error
+  -- Params: msg: error message reported by DP
+  -- example: { version = <latest version of deltas>, error = <flatten error>, }
+  manager.callbacks:register("kong.sync.v2.notify_validation_error", function(node_id, msg)
+    ngx_log(ngx_DEBUG, "[kong.sync.v2] received validation error")
+    -- TODO: We need a better error handling method, it might report this error
```

Comment:
Should we associate the error with a specific call?

Reply:
yes we could. but it might be a TODO ticket later. Need to have an idea how to recover this with our product manager.